### ISSUE TITLE
1.3.1 Play/ skip buttons on Mac keyboard dont work

### ISSUE THREAD

**Issue by roekone** [HermesApp/Hermes#302]
I just updated to 1.3.1 and now my keyboard controls dont work. Standard wired Mac key board. Imac running Sierra 10.12.3 (16D32)

**Comment by SinisterSmith87** [comment]
Here too on MacBook Pro mid 2009.

**Comment by atbe** [comment]
Confirming same issue after taking 1.3.1 update.

MAC: `MacBook Pro (Retina, 13-inch, Mid 2014)`
OS: `10.12.3 (16D32)`

**Comment by nriley** [comment]
OK, probably something to do with how Apple added a documented way to do this in 10.12.2 and then broke it in 10.12.3.  See if https://github.com/HermesApp/Hermes/issues/300#issuecomment-289442489 works for you?  I will definitely make another release to fix this once I get a chance.

**Comment by shaun10** [comment]
I can't use this either after upgrading. Is there a way to go backwards? I'm on 10.11.6 El Capitan

**Comment by roekone** [comment]
Confirmed.  Unchecking "Control playback with media keys" in Hermes preferences the re-checking it fixes the issue in 1.3.1

**Comment by roekone** [comment]
@shaun10 , use Time machine to restore the Hermes file in the "Applications" folder from yesterday's backup. If you use Time machine that is

**Comment by SinisterSmith87** [comment]
Uncheck and recheck works. Try it.

**Comment by bmehling** [comment]
Confirmed: Uncheck/Recheck of media control option resets. MBPro 13" w/ touchbar; 10.12.3 (16D32)

**Comment by Rayzr522** [comment]
Yep, same problem, unchecking and rechecking works like a charm.

**Comment by akiss20** [comment]
Checkbox fix worked on my 2016 touchbar macbook pro

**Comment by ZevEisenberg** [comment]
Me too. They work for a while, then stop working, seemingly after long pauses or idle periods? I'm on 10.13.2 (17C88).

**Comment by nriley** [comment]
Yeah unfortunately think this is an Apple bug. Not much we can do, maybe some day they'll provide a "real" API for background playback control on macOS like on iOS.  (It looked almost like that was going to happen with the Touch Bar changes but then they *undid* what they did a few point releases later.)

**Comment by nriley** [comment]
Sounds reasonable, but I'm not sure how often to "do the same thing"
and whether it requires that Hermes be brought to the front.  I'm not
aware of a Radar on this.

**Comment by ZevEisenberg** [comment]
Yep, unfortunately it'll be hard to test. Maybe you can simulate a media key press via Apple Events and set up some kind of long-running test to see how often it breaks?
